src/providers/index.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios"
+import jwtDecode from "jwt-decode"
+
+import buildCharacterProvider from "./character"
+import buildAuthenticationProvider from "./authentication"
+import buildStorageProvider from "./storage"
+import buildJwtProvider from "./jwt"
+
+const baseURL = "http://localhost:3050" // TODO: FIX TO USE ENV process.env.REACT_APP_MAHABHARATH_API
```

Comment:
I had some issues using the ENV variable and had to hard code the URL, if someone can debug and fix this I would be really happy.